src/core/layout/qgslayoutitemlegend.cpp

```diff
@@ -821,6 +875,11 @@ void QgsLayoutItemLegend::doUpdateFilterByMap()
   mForceResize = true;
 }
 
+QString QgsLayoutItemLegend::themeName() const
+{
+  return mThemeName;
```

Comment:
@nyalldawson , any reason why this isn't simply in the header file?

Reply:
There's no real benefit to it being there -- this isn't performance critical code

Reply:
Okeydokey. Merge away! :)